source/usage-examples/command.txt

```diff
@@ -4,20 +4,20 @@
 Run a Command
 =============
 
-You can run all raw database operations using the
-`db.command() <{+api+}/classes/Db.html#command>`__ method. Call the ``command()`` method with
-your command object on an instance of a database for diagnostic and
-administrative tasks such as fetching server stats or initializing a replica
-set.
-
-.. note::
-    Use the :mongosh:`MongoDB Shell </>` for administrative tasks instead of
-    the {+driver-short+} whenever possible.
-
-You can specify additional options in the ``options`` object passed in
-the second parameter of the ``command()`` method. For more information
-on the options you can pass, see the
-`db.command() API documentation <{+api+}/classes/Db.html#command>`__.
+You can run database operations by using the
+`Db.command() <{+api+}/classes/Db.html#command>`__ method. Call the
+``command()`` method with your command document on an instance of a
+database.
```

Comment:
Suggestion:

I think the second sentence attempts to fit too much information. I also think this could be more clear if broken down into communicating the following info in order:

1) The command() method allows you to run any MongoDB database operation. This is an instance method of the Db class.

2) To specify a command, create a command document that follows the command syntax (https://www.mongodb.com/docs/manual/reference/command/#command-syntax).

3) To run the command on the database, pass the command document to the command() instance method.

Let me know if you need any help crafting this description from this breakdown.

Reply:
Since this is a usage example, I dont think we should describe the command syntax. I think it is sufficient to mention that users need to create a command document and then link out to the manual

Reply:
I think we're in agreement. I meant was that this should describe the method syntax (which it currently does, but could be more clear), not the command syntax. The important part is to mention that it is a document that contains a command because that's the required parameter.